scripts/qa_crowbarsetup.sh

```diff
@@ -3093,6 +3093,15 @@ function deploy_single_proposal
 # apply all wanted proposals on crowbar admin node
 function onadmin_proposal
 {
+    local proposals
+    local default_proposals
+
+    default_proposals="
+        nfs_client pacemaker database rabbitmq keystone swift \
+        ceph glance cinder neutron nova `horizon_barclamp` ceilometer heat \
+        manila trove barbican magnum sahara murano tempest"
```

Comment:
I would remove "swift ceph manila trove" from default. May be something else too.
Just to keep the defaults to the minimum.